### PR TITLE
fix: remove cap limitation

### PR DESCRIPTION
**Pull request by sakulstra**
replaces https://github.com/bgd-labs/aave-helpers/pull/194

### CODE REVIEW

File: src/ProtocolV3TestBase.sol [hunk 1]
```diff
@@ -306,8 +305,7 @@ contract ProtocolV3TestBase is CommonTestBase {
    */
   function _getGoodCollateral(
     IPool pool,
-    ReserveConfig[] memory configs,
-    uint256 minSupplyCapDollarMargin
+    ReserveConfig[] memory configs
   ) private view returns (ReserveConfig memory config) {
```
Comment (Zer0dot) [inline src/ProtocolV3TestBase.sol]:
```suggestion
  ) private pure returns (ReserveConfig memory config) {
```

File: src/ProtocolV3TestBase.sol [hunk 2]
```diff
@@ -306,8 +305,7 @@ contract ProtocolV3TestBase is CommonTestBase {
    */
   function _getGoodCollateral(
     IPool pool,
```
Comment (Zer0dot) [inline src/ProtocolV3TestBase.sol]:
```suggestion
    IPool,
```

**Review by Zer0dot — APPROVED** [review]

File: src/ProtocolV3TestBase.sol (hunk 2 above)
Comment (sakulstra) [reply src/ProtocolV3TestBase.sol]: Ah in that case probably better to just remove completely

Comment (Zer0dot) [reply src/ProtocolV3TestBase.sol]: Sure, either works

**Review by Zer0dot — APPROVED** [review]

**Pull request merged**